tests/ord-service/tests/subscription_flow_test.go

```diff
@@ -38,102 +39,105 @@ import (
-
-	providedTenantIDs := tenantfetcher.Tenant{
-		TenantID:               accountID,
-		SubaccountID:           subscriptionConsumerSubaccountID,
-		Subdomain:              tenantfetcher.DefaultSubdomain,
-		SubscriptionProviderID: subscriptionProviderID,
-	}
-
-	tenantProperties := tenantfetcher.TenantIDProperties{
-		TenantIDProperty:               testConfig.TenantIDProperty,
-		SubaccountTenantIDProperty:     testConfig.SubaccountTenantIDProperty,
-		CustomerIDProperty:             testConfig.CustomerIDProperty,
-		SubdomainProperty:              testConfig.SubdomainProperty,
-		SubscriptionProviderIDProperty: testConfig.SubscriptionProviderIDProperty,
-	}
-
-	// Build a request for consumer subscription
-	request := tenantfetcher.CreateTenantRequest(t, providedTenantIDs, tenantProperties, http.MethodPut, testConfig.TenantFetcherFullRegionalURL, testConfig.ExternalServicesMockURL)
-
-	httpClient := &http.Client{
-		Timeout: 10 * time.Second,
-		Transport: &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-		},
-	}
-
-	t.Log(fmt.Sprintf("Creating a subscription between consumer with subaccount id: %s and provider with name: %s and subaccount id: %s", tenantfetcher.ActualTenantID(providedTenantIDs), runtime.Name, subscriptionProviderSubaccountID))
-	response, err := httpClient.Do(request)
-
-	require.NoError(t, err)
-	require.Equal(t, http.StatusOK, response.StatusCode)
-	tenantfetcher.AssertRuntimeSubscription(t, ctx, runtime.ID, providedTenantIDs, dexGraphQLClient, testConfig.ConsumerSubaccountIdsLabelKey)
-
-	// HTTP client configured with manually signed client certificate
-	extIssuerCertHttpClient := extIssuerCertClient(t, subscriptionProviderSubaccountID)
-
-	// Create a token with the necessary consumer claims and add it in authorization header
-	claims := map[string]interface{}{
-		"subsc-key-test": "subscription-flow",
-		"scope":          []string{},
-		"tenant":         subscriptionConsumerSubaccountID,
-		"identity":       "subscription-flow-identity",
-		"iss":            testConfig.ExternalServicesMockURL,
-		"exp":            time.Now().Unix() + int64(time.Minute.Seconds()),
-	}
-	headers := map[string][]string{"Authorization": {fmt.Sprintf("Bearer %s", token.FromExternalServicesMock(t, testConfig.ExternalServicesMockURL, claims))}}
-
-	// Make a request to the ORD service with http client containing certificate with provider information and token with the consumer data.
-	respBody := makeRequestWithHeaders(t, extIssuerCertHttpClient, testConfig.ORDExternalCertSecuredServiceURL+"/systemInstances?$format=json", headers)
-
-	require.Equal(t, 1, len(gjson.Get(respBody, "value").Array()))
-	require.Equal(t, consumerApp.Name, gjson.Get(respBody, "value.0.title").String())
+func TestSubscriptionFlow(stdT *testing.T) {
+	t := testingx.NewT(stdT)
+	t.Run("Consuming provider resources after successful subscription", func(t *testing.T) {
```

Comment:
This nesting is required in order to use the skip functionallity, provided by the new `t` we create.
All changed lines below are just identation.

Reply:
Just to note that this is the value that should be added in the regex, maybe we need to make it a bit more specific / unique.